test/k8sT/example.go

```diff
@@ -0,0 +1,17 @@
+package k8sT
```

Comment:
add copyright header if this file is going to be kept. I think this can be deleted.